mach-test/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["gnzlbg <gonzalobg88@gmail.com>"]
 build = "build.rs"
 
 [dependencies]
-mach = { path = ".." }
+mach = { path = "..", features = ["unstable"] }
```

Comment:
`ctest` does not white-list structs with wrong layouts anymore, so without `repr(packed)` it would fail. This should catch on CI that newer types introduced in the future have at least the proper layout on nightly.